.chloggen/pkg-stanza-rm-sugared.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: breaking
```

Comment:
Is changing the logging format a breaking change?

Reply:
Potentially, for users who are relying on semantic structure of logs. It's pretty minor but I figured in this case maybe it was worth noting because that change affect many log statements.